lib/cli/options.js

```diff
@@ -195,6 +195,11 @@ const loadPkgRc = (args = {}) => {
           `Unable to read/parse ${filepath}: ${err}`,
           filepath
         );
+      } else if (err.toString().includes('SyntaxError')) {
```

Comment:
If we want to separate the parsing from the reading errors, then I would suggest instead splitting this try/catch into an initial one for reading followed by a second one for parsing – that way we will not have to check the error as we know it's `JSON.parse()` that fails

Also gives more granular error for the existing use case – different error message depending on whether it can't read or can't parse

Reply:
Seems like a good idea.  Also, would it be useful to give an error for parsing errors more like the one npm gives, i.e. "make sure your package.json is JSON and not Javascript"?

Reply:
I don’t think it’s needed, just a warning saying “could not check package.json due to error while parsing” is probably the clearest

Reply:
I've simply split it into "Unable to read" and "Unable to parse" for the moment, not sure if there is a better wording.

Reply:
Looks great to me 🙂 I like it!